Framework/PythonInterface/test/python/mantid/plots/axesfunctionsTest.py

```diff
@@ -269,6 +274,19 @@ def test_1d_y_axes_label_distribution_workspace_auto_distribution_off(self):
         finally:
             config['graph1d.autodistribution'] = 'On'
 
+    def test_get_data_for_plot_binplot_binedgeaxis(self):
```

Comment:
:+1: for the new tests.

Reply:
Thanks. release note added. And fixed a typo in the commit message.